src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryColor.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.PowerFx.Core.IR;
+using Microsoft.PowerFx.Core.IR.Nodes;
+using Microsoft.PowerFx.Core.Types.Enums;
+using Microsoft.PowerFx.Interpreter;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Functions
+{
+    internal partial class Library
+    {
+        public static FormulaValue TextToColor(IRContext irContext, StringValue[] args)
```

Comment:
nit: we usually use AToB as the name for implicit coercion functions, and TextToColor is not one of our supported coercions. Given that this is the implementation of the ColorValue builtin, can we rename it to ColorValue?

Reply:
Makes sense. I renamed it to ColorValue, as that's the actual function... hope that is ok.